rdr_service/dao/genomics_dao.py

```diff
@@ -3602,15 +3731,44 @@ def get_aw3_array_records(self, **kwargs):
                 ParticipantSummary.withdrawalStatus == WithdrawalStatus.NOT_WITHDRAWN,
                 ParticipantSummary.suspensionStatus == SuspensionStatus.NOT_SUSPENDED,
                 GenomicAW3Raw.id.is_(None)
-            ).distinct().all()
+            )
```

Comment:
The query looks good. I tested it on prod and it took 20 seconds, which is acceptable considering it needs to scan the gc_data_file table. As we discussed, please remove the file-received flags.